packages/engine/paima-standalone/src/utils/import.ts

```diff
@@ -67,3 +68,13 @@ const PRECOMPILES_FILENAME = 'packaged/precompiles.cjs';
 export function importPrecompiles(): PreCompilesImport {
   return importFile<PreCompilesImport>(PRECOMPILES_FILENAME);
 }
+
+const EVENTS_FILENAME = 'packaged/events.cjs';
+export type AppEventsImport = { events: ReturnType<typeof generateAppEvents> };
+
+/**
+ * Reads repackaged user's code placed next to the executable in `events.cjs` file
+ */
+export function importEvents(): AppEventsImport {
+  return importFile<AppEventsImport>(EVENTS_FILENAME);
```

Comment:
Probably for both this and the precompiles import, we should give a default value if the import fails (otherwise, this is a tedious breaking change for every existing Paima dApp to handle)

Reply:
Makes sense, I guess having no events should be a valid choice anyway (maybe not for precompiles). Currently when I don't care about it I just do `touch packaged/precompiles.cjs`, but it's not ideal, and wouldn't work for events I think.